### ISSUE TITLE
Please add nojdk packages of Bazel for Windows

### ISSUE THREAD

**Issue by JayBazuzi** [bazelbuild/bazel#13779]
### Description of the problem / feature request:

Currently Bazel produces `nojdk` packages of Bazel for Linux and Mac, but not for Windows. Please expand that to include Windows.

![image](https://user-images.githubusercontent.com/1259628/127718627-13fbce89-6a94-4c34-a3e5-8e7a3684e0f5.png)

### Feature requests: what underlying problem are you trying to solve with this feature?

Bazel 4.1.0 comes with JDK 11.0.6 which has known security vulnerabilities. Our organization requires that we stay up to date with dependencies that have CVEs, and JDK 11.0.6 is flagged as prohibted.

We know that we can supply a JDK with `JAVA_HOME` and that Bazel will use it (which is great!) but the non-`nojdk` distribution of Bazel still unpacks the older JDK on the machine even though it doesn't use it, which is still flagged by our organization's security scans.

### What operating system are you running Bazel on?

Windows (and Linux and Mac, but this request is specific to Windows).

### What's the output of `bazel info release`?

```
INFO: Invocation ID: 10b6c3d7-b46a-4947-a546-279ba6c6e4a1
release 4.1.0
```

###  Have you found anything relevant by searching the web?

https://bazelbuild.slack.com/archives/CA31HN1T3/p1627277344349100

**Comment by meteorcloudy** [comment]
@comius Looks like we should also upgraded our embedded JDK version to not use `11.0.6` ?

**Comment by comius** [comment]
I tried upgrading jdk before, but I'm blocked by android, and nothing got resolved.
Related issues:
https://github.com/bazelbuild/bazel/issues/12512
https://github.com/bazelbuild/bazel/pull/13444
https://github.com/bazelbuild/bazel/issues/12880

**Comment by philwo** [comment]
Was this fixed by https://github.com/bazelbuild/continuous-integration/pull/1206?

**Comment by meteorcloudy** [comment]
![image](https://user-images.githubusercontent.com/4171702/131494611-84f81d7d-692c-472b-ab1a-bc8e313b3d7c.png)
Yes no_jdk binaries for Windows are available for 4.2.1 now.
